utils/article_filter.py

```
@@ -0,0 +1,13 @@
+def filter_condition(article, lang, topic):
+    # 언어 필터 적용
+    if lang is None or lang == article.get("lang"):
+        lang_filter = True
+    else:
+        lang_filter = False
```

Ruff (0.11.9)

Use ternary operator `lang_filter = True if lang is None or lang == article.get("lang") else False` instead of `if`-`else`-block

Replace `if`-`else`-block with `lang_filter = True if lang is None or lang == article.get("lang") else False`

(SIM108)